go/sqlescape/ids_test.go

```diff
@@ -26,12 +29,104 @@ func TestEscapeID(t *testing.T) {
 	}, {
 		in:  "a`a",
 		out: "`a``a`",
+	}, {
+		in:  "`fo`o`",
+		out: "```fo``o```",
+	}, {
+		in:  "",
+		out: "``",
 	}}
 	for _, tc := range testcases {
-		out := EscapeID(tc.in)
-		if out != tc.out {
-			t.Errorf("EscapeID(%s): %s, want %s", tc.in, out, tc.out)
-		}
+		t.Run(tc.in, func(t *testing.T) {
+			out := EscapeID(tc.in)
+			if out != tc.out {
+				assert.Equal(t, out, tc.out)
+			}
```

Comment:
This is superfluous. No point in asserting that something is true right after it's verified to be false. Plain `assert` is the way to go.
```suggestion
			assert.Equal(t, out, tc.out)
```

Reply:
Done